backend/api/student/simulation_instances.py

```
@@ -3,41 +3,281 @@
+        from sqlalchemy.orm import joinedload
+        
+        cohort_simulations_query = db.query(CohortSimulation).join(
+            Scenario, CohortSimulation.simulation_id == Scenario.id
+        ).join(
+            Cohort, CohortSimulation.cohort_id == Cohort.id
+        ).options(
+            joinedload(CohortSimulation.simulation),
+            joinedload(CohortSimulation.cohort).joinedload(Cohort.creator)
+        ).filter(
+            CohortSimulation.cohort_id.in_(cohort_ids),
+            Scenario.is_draft == False,  # Only published simulations
```

Ruff (0.13.3)

Avoid equality comparisons to `False`; use `not Scenario.is_draft:` for false checks

Replace with `not Scenario.is_draft`

(E712)